src/management/GuardianManager.js

```diff
@@ -280,4 +322,134 @@ utils.wrapPropertyMethod(GuardianManager, 'updateFactorTemplates', 'factorsTempl
  */
 utils.wrapPropertyMethod(GuardianManager, 'updateFactor', 'factors.update');
 
+/**
+ * Get enabled Guardian policies
+ *
+ * @method    getPolicies
+ * @memberOf  module:management.GuardianManager.prototype
+ *
+ * @example
+ * management.guardian.getPolicies(function (err, policies) {
+ *   console.log(policies);
+ * });
+ *
+ * @param   {Function}  [cb]              Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(GuardianManager, 'getPolicies', 'policies.get');
+
+/**
+ * Update enabled Guardian policies
+ *
+ * @method    updatePolicies
+ * @memberOf  module:management.GuardianManager.prototype
+ *
+ * @example
+ * management.guardian.updatePolicies({}, [
```

Comment:
The first argument here is the generic "params", that we have no use for. But since this is a convention for all GET endpoints, it seems that it is preferable to keep it here.

See also other methods that have the same approach:
- https://github.com/auth0/node-auth0/blob/48b7f8fa9d9ebd04928036436a415dc456b55cbb/src/management/PromptsManager.js#L56-L78
- https://github.com/auth0/node-auth0/blob/48b7f8fa9d9ebd04928036436a415dc456b55cbb/src/management/BrandingManager.js#L56-L78

Reply:
One argument to keep the `param` argument is to allow extending the method with additional parameters/options in the future (ex. if the API endpoint starts accepting optional query params).